File: 2756-buy-two-chocolates/2756-buy-two-chocolates.c
int buyChoco(int* prices, int pricesSize, int money){
  int minMoney = -1;
    int n = pricesSize;

    for (int i = 0; i < n-1; i++){
        for (int j = i+1; j < n; j++){
            int diff = money - prices[i] - prices [j];
            if (diff >= 0 && diff > minMoney){
                minMoney = diff;
            }
        }
    }

    if (minMoney == -1){
        return money;
    } 
    else{
        return minMoney;
    }
}

File: 2756-buy-two-chocolates/2756-buy-two-chocolates.js
/**
 * @param {number[]} prices
 * @param {number} money
 * @return {number}
 */
var buyChoco = function(prices, money) {
    var minMoney = -1;
    var n = prices.length;

    for (var i = 0; i < n-1; i++){
        for (var j = i+1; j < n; j++){
            var diff = money - prices[i] - prices [j];
            if (diff >= 0 && diff > minMoney){
                minMoney = diff;
            }
        }
    }

    if (minMoney === -1){
        return money;
    } 
    else{
        return minMoney;
    }

};